lib/core/presentation/pages/profile/profile_page.dart

```diff
@@ -1,109 +1,16 @@
-import 'package:app/gen/assets.gen.dart';
+import 'package:app/core/presentation/pages/profile/views/profile_page_view.dart';
 import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
 
 @RoutePage()
 class ProfilePage extends StatelessWidget {
-  const ProfilePage({super.key});
+  final String userId;
+  const ProfilePage({super.key, @PathParam("id") required this.userId});
 
   @override
   Widget build(BuildContext context) {
-    return AuthGuard(
-      unauthenticatedBuilder: (context) => const LoginPage(),
-      authenticatedBuilder: (context) => const ProfilePageView(),
-    );
-  }
-}
-
-class ProfilePageView extends StatefulWidget {
-  const ProfilePageView({super.key});
-
-  @override
-  State<ProfilePageView> createState() => _ProfilePageViewState();
-}
-
-class _ProfilePageViewState extends State<ProfilePageView> with SingleTickerProviderStateMixin {
-  double get _headerHeight => 230;
-
-  int get _tabCount => 6;
-
-  late final TabController _tabCtrl = TabController(
-    length: _tabCount,
-    vsync: this,
-    initialIndex: 0,
-  );
-
-  @override
-  void dispose() {
-    _tabCtrl.dispose();
-    super.dispose();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    final colorScheme = Theme.of(context).colorScheme;
-    return Scaffold(
-      backgroundColor: colorScheme.primary,
-      appBar: _appBar(colorScheme),
-      body: NestedScrollView(
-        headerSliverBuilder: (context, innerBoxIsScrolled) => [
-          SliverAppBar(
-            floating: true,
-            expandedHeight: _headerHeight,
-            collapsedHeight: _headerHeight,
-            flexibleSpace: ProfilePageHeader(),
-            forceElevated: innerBoxIsScrolled,
-          ),
-          SliverOverlapAbsorber(
-            handle: NestedScrollView.sliverOverlapAbsorberHandleFor(context),
-            sliver: SliverPersistentHeader(
-              pinned: true,
-              floating: false,
-              delegate: ProfileTabBarDelegate(controller: _tabCtrl),
-            ),
-          ),
-        ],
-        body: TabBarView(
-          controller: _tabCtrl,
-          children: [
-            EmptyTabView(),
-            ProfileCollectibleTabView(),
-            ProfileEventTabView(),
-            ProfilePhotosTabView(),
-            EmptyTabView(),
-            ProfileInfoTabView(),
-          ],
-        ),
-      ),
-    );
-  }
-
-  LemonAppBar _appBar(ColorScheme colorScheme) {
-    return LemonAppBar(
-      title: '@coop.er',
-      leading: BurgerMenu(),
-      actions: [
-        ThemeSvgIcon(
-          color: colorScheme.onSurface,
-          builder: (filter) => Assets.icons.icShare.svg(colorFilter: filter),
-        ),
-        ThemeSvgIcon(
-          color: colorScheme.onSurface,
-          builder: (filter) => Assets.icons.icMoreHoriz.svg(colorFilter: filter),
-        ),
-      ],
+    return ProfilePageView(
+      userId: '',
```

Comment:
Is it anything missing here

Reply:
ah no, this will be used for future view other user profile. 
For this feature, MyProfilePageView is being used.